dashboard.go

```diff
@@ -141,6 +141,9 @@ type Chart struct {
 
 	// ChartSettings are custom settings for the chart
 	ChartSettings ChartSetting `json:"chartSettings"`
+
+	// ChartSettings are custom settings for the chart
```

Comment:
Could you just change this to reflect ChartAttributes?

Reply:
Oops, yeah will do.

Reply:
done